Babylon/groups/working_dir/__init__.py

```diff
@@ -3,15 +3,14 @@
 
 from .commands import list_commands
 from .groups import list_groups
-from ...utils.decorators import pass_environment
+from ...utils.environment import Environment
 
 
 @group()
-@pass_environment
 @pass_context
-def working_dir(ctx, environment):
+def working_dir(ctx):
     """Command group handling working directory information"""
-    ctx.obj = environment.working_dir
+    ctx.obj = Environment().working_dir
```

Comment:
Probably unnecessary !

Reply:
Yeah probably forgot to remove those ^^'